test/potluck_test.rb

```diff
@@ -0,0 +1,102 @@
+require './lib/dish'
+require './lib/potluck'
+require 'minitest/autorun'
+require 'minitest/pride'
+require 'pry'
+
+class PotluckTest < Minitest::Test
+
+  def test_existence
+    potluck = Potluck.new("7-13-18")
+    assert_instance_of Potluck, potluck
+  end
+
+  def test_attributes
+    potluck = Potluck.new("7-13-18")
+    result = potluck.date
+    expected = "7-13-18"
+    assert_equal expected, result
+    result = potluck.dishes
+    expected = []
+    assert_equal expected, result
+  end
+
+  def test_it_can_hold_dishes
+    potluck = Potluck.new("7-13-18")
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+    cocktail_meatballs = Dish.new("Cocktail Meatballs", :entre)
+    potluck.add_dish(couscous_salad)
+    potluck.add_dish(cocktail_meatballs)
+    result = potluck.dishes
+    expected = [couscous_salad, cocktail_meatballs]
+    assert_equal expected, result
+  end
```

Comment:
Might consider breaking this up with some whitespace for readability. Consider separating into sections: setup, execution, assertions (each if needed).